### ISSUE TITLE
Warning about Atom modifying Key system DLL

### ISSUE THREAD

**Issue by sublines** [atom/atom#8263]
Does this look like something Atom is supposed to  do or is my atom hijacked? 
The number 23 next to the word block is just a countdown in case the user does not select an action.
![unbenannt-1](https://cloud.githubusercontent.com/assets/2164222/9132387/71b57538-3cf3-11e5-89dc-989f8b074079.jpg)

**Comment by 50Wliu** [comment]
@sublines does this still happen with Atom 1.0.5?

**Comment by sublines** [comment]
The shortcut I used to open Atom did indeed open the older version 1.0.2. 
I have updated the shortcut and didn't get the message since. 
However, this did not happen every time that I opened Atom, so I can not say if that was the cause. 

By the way: On Windows, Atom is saved on C:\Users\username\AppData\Local\atom
In that folder I have the versions 1.0.2, 1.0.3 and 1.0.5 with all subfolders and files. 
Should Atom really keep all past versions stored on the computer and is there a way to have ashortcut to a file that always opens the latest version?
